services/github/client.go

```diff
@@ -0,0 +1,37 @@
+package github
+
+import (
+	"context"
+
+	"github.com/google/go-github/v45/github"
+
+	"github.com/Scalingo/go-scalingo/v4/debug"
+)
+
+const (
+	owner = "scalingo"
+	repo  = "cli"
+)
+
+type client struct {
+	githubRepoService *github.RepositoriesService
+}
+
+func (c client) ListReleases(ctx context.Context, opts *github.ListOptions) ([]*github.RepositoryRelease, error) {
+	repoReleases, githubResponse, error := c.githubRepoService.ListReleases(ctx, owner, repo, opts)
+	debug.Printf("github response: %#v\n", githubResponse)
+	return repoReleases, error
```

Comment:
You should annotate the error and rename it to `err`, like we do everywhere 😅

Reply:
oops :sweat_smile: